decidim-results/app/commands/decidim/results/admin/update_result.rb

```diff
@@ -31,30 +31,38 @@ def call
 
         private
 
+        attr_reader :result, :form
+
         def update_result
-          @result.update_attributes!(
-            scope: @form.scope,
-            category: @form.category,
-            title: @form.title,
-            short_description: @form.short_description,
-            description: @form.description
+          result.update_attributes!(
+            scope: form.scope,
+            category: form.category,
+            title: form.title,
+            short_description: form.short_description,
+            description: form.description
           )
         end
 
         def proposals
-          @result.sibling_scope(:proposals).where(id: @form.proposal_ids)
+          @proposals ||= result.sibling_scope(:proposals).where(id: form.proposal_ids)
+        end
+
+        def meeting_ids
```

Comment:
I thought about this, but it didn't seem to fit in the form in my opinion. Can be easily moved though.